R/layout.R

```diff
@@ -158,7 +157,6 @@ config <- function(p, ..., cloud = FALSE, locale = NULL, mathjax = NULL) {
   args <- list(...)
   if ("collaborate" %in% names(args)) warning("The collaborate button is no longer supported")
   p$x$config <- modify_list(p$x$config, args)
-  p$x$config$cloud <- cloud
 
```

Comment:
To maintain backwards compatibility, keep the arg, deprecate `cloud = FALSE`. Add argument `sendDataToCloud = cloud`.

Eventually remove both